lib/script.js

```diff
@@ -105,6 +109,30 @@ Script.prototype[onRpcMessage] = function (id, operation, params, data) {
   }
 };
 
+Script.prototype[onLog] = function(level, text) {
+  switch (level) {
+    case 'info':
+      console.log(text);
+      break;
+    case 'warning':
+      console.warn(text);
+      break;
+    case 'error':
+      console.error(text);
+      break;
+  }
+};
+
+Script.prototype.setLogHandler = function (handler) {
+  if (typeof handler === 'undefined') {
```

Comment:
I'd simplify this to:

```js
  this[logHandler] = (handler !== null) ? handler.bind(this) : log;
```

I tend to avoid sanity-checking arguments, as it's better to leave this to the TypeScript compiler and avoid paying for this at runtime. (At some point it would be good to rewrite frida-node in TypeScript, or at least provide type definitions.)